routes.py

```diff
@@ -0,0 +1,114 @@
+from app import app, bp, db
+from models import Usercredential, Habit
+import os
+import json
+import requests
+import flask
+from flask_login import login_user, current_user, LoginManager
+from flask_login.utils import login_required
+from datetime import date
+import base64
+
+
+@bp.route('/index')
+# @login_required
+def index():
+    # TODO: insert the data fetched by your app main page here as a JSON
+    DATA = {"your": "data here"}
+    data = json.dumps(DATA)
+    return flask.render_template(
+        "index.html",
+        data=data,
+    )
+
+
+@bp.route('/save', methods=["POST"])
+def saveTestHabit():
+    response_json = flask.request.json
+    print(response_json)
+    title = response_json['title']
+    category = response_json['category']
+    date_created = date.today()
+    target_days = response_json['target_days']
+
+    # habit = Habit(
+    #     user=10,  # hardcoded user id for test purposes
+    #     title=response_json['title'],
+    #     category=response_json['category'],
+    #     date_created=date.today(),
+    #     target_days=response_json['target_days'],
+    # )
+
+    db.session.add(habit)
+    db.session.commit()
+
+    # change this to something more meaningful
+    return flask.jsonify({"status": 'success'})
+
```

Comment:
Would you mind deleting this and replacing with `...` until i push my code that touches this function?